Bottom: 69b165e0f42557270e814f2846ac9929b97498b3
Top:    901d22538f8645e8c4fae829e57ff3c4a67348dc
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-05-01 01:58:46 +0100

Refresh of adjust-step-export-to-2-3mm

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 850a011..7ec84c0 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -59,7 +59,7 @@
 #endif
 
 
-#define HACK_BOARD_THICKNESS MM_TO_COORD(1.6)
+#define HACK_BOARD_THICKNESS MM_TO_COORD(2.3)
 
 static GList *object3d_test_objects = NULL;
